packages-new/components/flag/src/components/osds-flag/constants/flag-iso-code.ts

```diff
@@ -0,0 +1,261 @@
+enum ODS_FLAG_ISO_CODE {
```

Comment:
I know we can't extend enum, but we shouldn't duplicate all the codes here.
You can use a normal JS object in this case, something like:
```
// your iso list from core
enum ISO {
  en = 'EN',
  fr = 'FR',
}

// your flag constant code
const FLAG = {
  ...ISO,
  EU: 'EU',
}

console.log(ISO) // Object { en: "EN", fr: "FR" }
console.log(FLAG) // Object { en: "EN", fr: "FR", EU: "EU" }
```

Reply:
I did it, but I'm not convinced by the lost of the typage and we lost the type of ODS_COUNTRY_ISO_CODE in the phone-number too